controller/article-controller.js

```diff
@@ -16,4 +16,18 @@ exports.getArticles = (req, res, next) => {
             res.status(200).send({articles})
         })
         .catch(next)
+}
+
+exports.getCommentsByArticleId = (req, res, next) => {
+    const {article_id} = req.params
+    const promises = [selectCommentsByArticleId(article_id)]
+    if (article_id){
+        promises.push(checkArticleExists(article_id))
+    }
+    Promise.all(promises)
+        .then((resolvedPromises) => {
+            const comment = resolvedPromises[0]
+            res.status(200).send(comment)
```

Comment:
This is the tiniest thing - but as it's an array of comments, I would pluralise the word comment.